node/src/BaseClient.ts

```diff
@@ -476,6 +521,14 @@ export class BaseClient {
      *
      * @param key - The key to increment its value.
      * @returns the value of `key` after the increment.
+     *
+     * @example
+     * ```typescript
+     * // Example usage of incr method to increment the value of a key
+     * await client.set("my_counter", "10");
```

Comment:
You can omit `set` step here (and in all docs too)

Reply:
I like it, I think its more informative